src/java.base/share/classes/java/lang/Process.java

```diff
@@ -106,7 +125,13 @@ public Process() {}
      * then this method will return a
      * <a href="ProcessBuilder.html#redirect-input">null output stream</a>.
      *
-     * <p>Implementation note: It is a good idea for the returned
+     * @apiNote
+     * Use {@link #getOutputStream} and {@link #outputWriter} with extreme care.
+     * Output to the {@code BufferedWriter} may be held in the buffer until
+     * {@linkplain BufferedWriter#flush flush} is called.
```

Comment:
I think this will need a bit of wordsmithing to make it clear that its the usage of both streams for the same Process requires great care (as it stands, it reads like the usage of either method is dangerous).